index.html

```diff
@@ -7,6 +7,8 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
   <title>Homepage| ACES opensource | Easily build your Webpages</title>
   <link rel="stylesheet" href="assets/css/home.css" />
+  <!-- my css -->
+  <link rel="stylesheet" href="./assets/css/-- css/my.css">
```

Comment:
Good contribution but thought you would have used home.css instead of creating a new file